languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -101590,6 +101590,30 @@ The accident victim died from her injuries.
             <example correction="">This <marker>is</marker> right.</example>
             <example>This <marker>is</marker> conforming with my current knowledge.</example>
         </rule>
+        <rule id="ITS_TO_IT_S_2" name="its -> it's [2]">
```

Comment:
These could be subrules of IT_IS_2, I think.

In any case, the previous commit on this topic and this one should be in the same rulegroup. You could call it "ITS_IT_IS". The first rule in it would automatically be designated ITS_IT_IS[1], the second one ITS_IT_IS[2], etc.